### PR TITLE
15 setup course listing students

### PR DESCRIPTION
**Pull request by OmarSelim01**
Setup the add member popup as well as the create team popup in the same page for both instructors and students, css overlapping issue for both popups resolved as well

### CODE REVIEW

**Review by Theos1010 — APPROVED** [review]

File: CodebustersPAS-ClientApp/src/pages/Teacher/CreateTeam/TeamOverview.tsx
Comment (LouisChoiniere) [inline CodebustersPAS-ClientApp/src/pages/Teacher/CreateTeam/TeamOverview.tsx:1]: Why did you commit an empty file?

**Review by LouisChoiniere — CHANGES_REQUESTED** [review]
Fix comment otherwise looks good

**Review by LouisChoiniere — APPROVED** [review]
lgtm

**Review by Theos1010 — COMMENTED** [review]
@OmarSelim01 When I run from your branch, on the Login page the "Don't have an account?" text isn't getting displayed anymore. Same happens with the "Already have an account?" on the Register page.

**Pull request merged**